bigtable-hbase-1.x-parent/bigtable-hbase-1.x/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableConnection.java

```diff
@@ -43,8 +43,27 @@ public void testBigtableConnectionExists() {
   }
 
   @Test
-  public void testTable() throws IOException {
+  public void testConfig_Basic() {
     Configuration conf = BigtableConfiguration.configure("projectId", "instanceId");
+    Assert.assertEquals("projectId", conf.get(BigtableOptionsFactory.PROJECT_ID_KEY));
+    Assert.assertEquals("instanceId", conf.get(BigtableOptionsFactory.INSTANCE_ID_KEY));
+    Assert.assertEquals(BigtableConfiguration.getConnectionClass().getName(), conf.get("hbase.client.connection.impl"));
+    Assert.assertNull(conf.get(BigtableOptionsFactory.APP_PROFILE_ID_KEY));
+  }
+
+  @Test
+  public void testConfig_AppProfile() {
+    Configuration conf = BigtableConfiguration.configure("projectId", "instanceId", "appProfileId");
+    Assert.assertEquals(conf.get(BigtableOptionsFactory.PROJECT_ID_KEY), "projectId");
+    Assert.assertEquals(conf.get(BigtableOptionsFactory.INSTANCE_ID_KEY), "instanceId");
+    Assert.assertEquals(conf.get(BigtableOptionsFactory.APP_PROFILE_ID_KEY), "appProfileId");
+    Assert.assertEquals(conf.get("hbase.client.connection.impl"),
+        BigtableConfiguration.getConnectionClass().getName());
+  }
+
+  @Test
+  public void testTable() throws IOException {
+    Configuration conf = BigtableConfiguration.configure("projectId", "instanceId", "appProfileId");
```

Comment:
Do we need to include appProfileId here?

Reply:
no, it's not needed